wp-content/themes/reconasia/assets/_scss/components/_post-meta.scss

```diff
@@ -1,52 +1,68 @@
 @use 'abstracts' as *;
 
 .post-meta {
-  color: var(--post-meta);
```

Comment:
Reminder that we should be using CSS variables for this. If you do a git rebase and look at my code for the page headers, you'll see how I used it to change the colors of the author & date information. That keeps it very reusable no matter where it's used on the site.